tests/publicLinkTest.js

```diff
@@ -97,7 +106,88 @@ describe('oc.publicFiles', function () {
         // CREATED SHARES
         let testFolderShare = null
 
-        beforeEach(function (done) {
+        beforeAll(async function (done) {
+          const promises = []
+          promises.push(setGeneralInteractions(provider))
+          promises.push(provider.addInteraction({
+            uponReceiving: 'create a public link share',
+            withRequest: {
+              method: 'POST',
+              path: Pact.Matchers.regex({
+                matcher: '.*\\/ocs\\/v(1|2)\\.php\\/apps\\/files_sharing\\/api\\/v1\\/shares',
+                generate: '/ocs/v1.php/apps/files_sharing/api/v1/shares'
+              }),
+              headers: validAuthHeaders
+              // body: 'shareType=3&path=%2F' + config.testFolder
```

Comment:
I guess that you do not [want|need] to be specific about the body when matching here.
Do you want to leave this as a comment?